internal/pkg/build/build.go

```diff
@@ -322,12 +328,12 @@ func getcp(def types.Definition, libraryURL, authToken string) (ConveyorPacker,
 }
 
 // MakeDef gets a definition object from a spec
-func MakeDef(spec string, remote bool) (types.Definition, error) {
-	return makeDef(spec, remote)
+func MakeDef(spec string) (types.Definition, error) {
+	return makeDef(spec)
 }
 
 // makeDef gets a definition object from a spec
-func makeDef(spec string, remote bool) (types.Definition, error) {
+func makeDef(spec string) (types.Definition, error) {
```

Comment:
I knew it is outside the scope of your PR but why have both `MakeDef` and `makeDef` since `MakeDef` simply directly call `makedef`?

Reply:
will make them public

Reply:
Looks like these changes have been made, so I'm resolving the convo.  Holler at me if I'm wrong.